### PR TITLE
Do not animate 'display' or the 'animation*' properties.

### PR DESCRIPTION
**Pull request by ewilligers**

### CODE REVIEW

File: src/normalize-keyframes.js [hunk 1]
```diff
@@ -150,6 +150,10 @@
     return value;
   }
 
+  function isPermittedKeyframeProperty(property) {
+    return property !== 'display' && property.lastIndexOf('animation', 0) === -1;
```
Comment (alancutter) [inline src/normalize-keyframes.js]: Why not indexOf()?

File: test/js/keyframe-effect-constructor.js [hunk 2]
```diff
@@ -354,4 +354,29 @@ suite('keyframe-effect-constructor', function() {
     effect.timing.iterations = 2;
     assert.closeTo(Number(getComputedStyle(target).opacity), 0.75, 0.01, 't=125 after setting iterations');
   });
+
+  test('display and animation* properties are not animated', function() {
+    var target = document.createElement('div');
+    target.style.position = 'absolute';
```
Comment (alancutter) [inline test/js/keyframe-effect-constructor.js]: No need to set this.

**Comment by alancutter** [comment]
LGTM with nits.

The description should reference the bug it's fixing.

File: src/normalize-keyframes.js (hunk 1 above)
Comment (ewilligers) [inline src/normalize-keyframes.js]: lastIndexOf(, 0) searches backwards from 0, meaning only one offset is tested. If the property name is long, indexOf needs to check many offsets.

**Comment by alancutter** [comment]
A test fails.

File: src/normalize-keyframes.js (hunk 1 above)
Comment (alancutter) [inline src/normalize-keyframes.js]: Ah missed the `, 0`.

**Comment by ewilligers** [comment]
> A test fails.
Now fixed.

**Pull request merged**